### PR TITLE
Rmsecl volumetrics

### PR DESCRIPTION
**Pull request by berland**
Finally a tool for comparing Eclipse and RMS volumetrics. 

Not sure if the naming is good.

This might primarily be a tool that is used in Webviz, but should possibly be a command line utility also.

Suggest to release this in komodo 2021.07 but considering limited news coverage before it hits Webviz.

### CODE REVIEW

**Comment by berland** [comment]
Depends on https://github.com/equinor/fmu-tools/pull/143

File: docs/scripts/rmsecl_volumetrics.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+RMSECL_VOLUMETRICS
```
Comment (jcrivenaes) [inline docs/scripts/rmsecl_volumetrics.rst:2]: I think we need a more concrete example in the documentation

Comment (berland) [reply docs/scripts/rmsecl_volumetrics.rst:2]: Now added.

Comment (jcrivenaes) [reply docs/scripts/rmsecl_volumetrics.rst:2]: Excellent!

**Review by jcrivenaes — APPROVED** [review]
LGTM!

**Pull request merged**